src/servers/src/http.rs

```diff
@@ -445,6 +448,15 @@ impl HttpServer {
             );
         }
 
+        // mem profiler
+        #[cfg(feature = "mem-prof")]
+        {
+            router = router.nest(
+                &format!("/{HTTP_API_VERSION}/prof"),
+                Router::new().route("/mem", routing::get(crate::http::mem_prof::mem_prof)),
+            );
+        }
```

Comment:
How about also route this path when `#[cfg(not(feature= "mem-prof"))]`, but return the message that it's not enabled?

Reply:
> How about also route this path when `#[cfg(not(feature= "mem-prof"))]`, but return the message that it's not enabled?

Currently if binary was not built with "mem-prof" feature, a 404 will be responded, I think it would be enough.